### PR TITLE
[FCE-937] Fix expo doctor issues

### PR DESCRIPTION
**Pull request by maksg**
## Description

Fixed warnings which appeared when running expo doctor

## Motivation and Context

Expo doesn't recommend using different package versions than those used by Expo, it may cause issues along the way.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by mironiasty** [comment]
WDYT about somewhere info about how to use expo doctor? Maybe in `CONTRIBUTING.md`?

File: examples/fishjam-chat/package.json [hunk 1]
```diff
@@ -15,20 +15,20 @@
   "dependencies": {
     "@fishjam-cloud/react-native-client": "*",
     "@gorhom/bottom-sheet": "^5.0.6",
-    "@react-native-async-storage/async-storage": "2.1.0",
+    "@react-native-async-storage/async-storage": "1.23.1",
     "@react-navigation/bottom-tabs": "^6.6.1",
     "@react-navigation/native": "^6.1.18",
     "@react-navigation/native-stack": "^6.11.0",
-    "expo": "^52.0.8",
+    "expo": "~52.0.11",
```
Comment (MiloszFilimowski) [inline examples/fishjam-chat/package.json:22]: **praise:** 👍

**Review by MiloszFilimowski — APPROVED** [review]

**Comment by maksg** [comment]
@mironiasty I've added the info about expo doctor to `CONTRIBUTING.md`

Also I think we should consider changing `dependabot.yml` to only update Expo and it's dependencies because otherwise the expo doctor will throw warnings again.

**Comment by maksg** [comment]
@mironiasty @MiloszFilimowski I've added expo dependencies to `ignore` in dependabot config so you can check them and say if you have any objections

**Review by MiloszFilimowski — APPROVED** [review]
Looks good!

File: .github/dependabot.yml [hunk 2]
```diff
@@ -11,3 +11,9 @@ updates:
     groups:
       development-dependencies:
         dependency-type: "development"
+    ignore:
+      - dependency-name: "@react-native-async-storage/async-storage"
+      - dependency-name: "expo"
+      - dependency-name: "react-native"
+      - dependency-name: "react-native-safe-area-context"
+      - dependency-name: "react-native-screens"
```
Comment (mironiasty) [inline .github/dependabot.yml:19]: Curious - can we ignore only `major` (and maybe `minor`) version for some of these dependencies? (with `update-types` [option](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#specifying-dependencies-and-versions-to-ignore))

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -92,6 +92,7 @@ When you're sending a pull request:
 
 - Prefer small pull requests focused on one change.
 - Verify that linters and tests are passing.
+- Run `npx expo-doctor` and make sure that your changes don't break any of it's checks.
```
Comment (mironiasty) [inline CONTRIBUTING.md:95]: At some point it might be worth to add this as CI check?

**Review by mironiasty — APPROVED** [review]

File: .github/dependabot.yml (hunk 2 above)
Comment (maksg) [reply .github/dependabot.yml:19]: We could but wouldn't that break expo doctor passing anyway?

File: CONTRIBUTING.md (hunk 3 above)
Comment (maksg) [reply CONTRIBUTING.md:95]: Agreed. I have created FCE-959 for that

**Comment by maksg** [comment]
@mironiasty I've added only `major` and `minor` as ignored types

**Pull request merged**